README.md

```diff
@@ -51,40 +76,36 @@ The `environment` initialisation option tells the client which API endpoint to m
 
 The `authentication` initialisation options provides information about the context the client's being initialised in, specifically the authentication information for the currently logged in customer - see "Authentication" below.
 
-### Authentication
-Because the Submarine Customer API is returning sensitive customer information (a list of their stored payment methods, saved subscriptions, and the contents of those subscriptions), authentication in required to retrieve or update any customer information.
+## 1.2 Authentication
+`submarine.js` using [Jason Web Tokens (JWTs)](https://jwt.io/) for authenticating client side requests, and returning sensitive customer information via the JWTs encoded `tokens`.
 
-Requests to the API are authenticated by providing three parameters in the querystring of all HTTP requests (whether `GET` or `POST` requests):
+This allows for a secure environment to exchange customer, and shop information flow via the client.
 
-* `shop` - the Shopify domain of the current store, eg `example.myshopify.com`;
-* `timestamp` - a UNIX timestamp of when the request was generated;
-* `signature` - a SHA256 HMAC signature generated from the ID of the logged in customer, the timestamp value, and a secret key made available to your theme via a shop-level metafield `shop.metafields.submarine.customer_api_secret`.
-
-For Shopify themes, these values should be generated within your Liquid templates and passed during initialisation. For other clients, such as mobile apps, these values should be generated within your application code before making calls.
-
-Here's an example of how you can initialised the Submarine client library within a Liquid template in your Shopify theme:
+Here's an example of how you can initialised the Submarine client library within a Liquid template in your **Shopify** theme:
 
 ```liquid
-<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/submarine-js@0.4.0-beta.2/dist/submarine.js"></script>
+    <script src="{{ 'submarine.js' | asset_url }}"></script>
```

Comment:
Keep the CDN JS reference please - we don’t need people to upload the latest version of `submarine.js` to their theme